### ISSUE TITLE
Better way for detecting changes in partner repo

### ISSUE THREAD

**Issue by oeway** [bioimage-io/collection-bioimage-io#437]
[The current way of forcing an update in the partner repo](https://github.com/imjoy-team/bioimage-io-resources/issues/10#issuecomment-1121420106) requires making changes to the collection repo, and also confusing  (e.g. I spent lots of time to figure out why the icons are not changed).

As I understand, it currently only detect changes from the manifest file, can we use the head commit hash instead?

**Comment by FynnBe** [comment]
yes, we probably should. the idea back then was that the `version` field could be used in the collection RDF entries to indicate changes. the bumping of the `version` field would imply a change in the file hash, which would be detected.
I suppose this workflow does not hold up to the reality of us ignoring RDF versions. 

Maybe my initial answer to "how to trigger a partner collection reevaluation" could have been "bump the appropriate `version` fields". However I think going with the commit hash will be more consistent. 

The problem with using the commit hash is that I the only way I see to get the commit hash is to checkout the partner collection repo; while we currently only need to download the partner collection file (and then selectively linked files during testing).

I'll try to avoid downloading the whole partner repo with something like:
`git clone <repo> --filter=blob:none --filter=tree:0 --no-checkout` --https://stackoverflow.com/questions/59276829/how-to-git-clone-without-downloading-the-file-objects

**Comment by oeway** [comment]
Maybe try the GitHub api, which required 1 http request: https://stackoverflow.com/a/45727280
